subprojects/docs/src/docs/userguide/running-builds/introduction/plugin_basics.adoc

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[plugin_basics]]
+= Plugin Basics
+
+Plugins are used to *extend build capability and customize Gradle*.
+
+image::gradle-basic-6.png[]
+
+Most features, like the ability to compile Java code, are added by _plugins_.
+
+Using plugins is also the primary mechanism for organizing build logic.
+
+Plugins are essentially a set of all useful tasks, such as compiling tasks, creating documentation, setting up source files, etc.
+
+*Applying* a plugin to a project allows the plugin to extend the project's and Gradle's capabilities.
+
+For example, The Spring Boot Gradle Plugin, `org.springframework.boot`, provides Spring Boot support.
+The Google Services Gradle Plugin, `com.google.gms:google-services`, enables Google APIs or Firebase services in your Android application.
+The Palantir Git-Version Plugin, `com.palantir.git-version`, is a plugin that generates a project version for use with Gradle by calling `git-describe`.
+
+== Plugin distribution
+
+Plugins are distributed in three ways:
+
+1. **Core plugins** - Gradle develops and maintains a set of <<plugin_reference#plugin_reference,Core Plugins>>.
+2. **Community plugins** - Gradle's community shares plugins via the https://plugins.gradle.org[Gradle Plugin Portal].
+3. **Local plugins** - Gradle enables user to create custom plugins using link:{groovyDslPath}/org.gradle.api.tasks.javadoc.Javadoc.html[APIs].
```

Comment:
This last link doesn't seem right. It links to the `Javadoc` task API. Should we like to the `Plugin` class API?